### PR TITLE
Insta feed

### PR DESCRIPTION
**Pull request by azsidhu**

### CODE REVIEW

File: digital_benchmark/digital_benchmark/settings.py [hunk 1]
```diff
@@ -31,12 +31,14 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'instagramBenchmark',
```
Comment (ehmadzubair) [inline digital_benchmark/digital_benchmark/settings.py]: snake_case it

File: digital_benchmark/digital_benchmark/urls.py [hunk 2]
```diff
@@ -19,4 +19,5 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('facebook_benchmark/', include('facebook_benchmark.urls')),
+    path('instagramBenchmark/', include('instagramBenchmark.urls'))
```
Comment (ehmadzubair) [inline digital_benchmark/digital_benchmark/urls.py]: instagram-benchmark

File: digital_benchmark/instagramBenchmark/templates/auth.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{% load static%}
+<link rel='stylesheet' type='text/css' href="{% static '/style.css' %}"/>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+
+<script>
+    function instaLogin() {
+            url="https://api.instagram.com/oauth/authorize/?client_id=4d8f538893ba481f88c0614865dc9310&redirect_uri=http://127.0.0.1:8000/instagramBenchmark/instaLogin&response_type=token"
```
Comment (ehmadzubair) [inline digital_benchmark/instagramBenchmark/templates/auth.html]: Use a Django Form to do this.

File: digital_benchmark/instagramBenchmark/templates/firstpage.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{% load static%}
+<link rel='stylesheet' type='text/css' href="{% static '/style.css' %}"/>
+
+<div id="loginDiv">
+    <p id="successText">Successfully loged in!</p>
```
Comment (ehmadzubair) [inline digital_benchmark/instagramBenchmark/templates/firstpage.html]: logged*

File: digital_benchmark/instagramBenchmark/urls.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from . import views
+
+app_name = 'instagramBenchmark'
+
+urlpatterns = [
+    path('', views.index, name='index'),
+    path('auth', views.auth, name='auth'),
+    path('instaLogin', views.instaLogin, name='instaLogin')
```
Comment (ehmadzubair) [inline digital_benchmark/instagramBenchmark/urls.py]: insta-login

File: digital_benchmark/instagramBenchmark/views.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+
+# Create your views here.
+def index(request):
+    return HttpResponse('instagram benchmark app is up!')
+
+def auth(request):
```
Comment (ehmadzubair) [inline digital_benchmark/instagramBenchmark/views.py]: create Class based Views

**Comment by ehmadzubair** [comment]
@azsidhu Add .pyc files into .gitignore

File: digital_benchmark/instagramBenchmark/templates/auth.html (hunk 3 above)
Comment (azsidhu) [reply digital_benchmark/instagramBenchmark/templates/auth.html]: This link is taking user to login screen provided by Instagram API, we are just using a single button to go to that link. Do we still need a django form??

**Comment by ehmadzubair** [comment]
@azsidhu merge this asap

**Pull request merged**